app/javascript/articles/components/CommentListItem.jsx

```diff
@@ -0,0 +1,67 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+/* global timeAgo */
+
+function createMarkup(comment) {
+  return { __html: comment.safe_processed_html };
```

Comment:
It's OK to inline this as the value for the `dangerouslySetInnerHTML` prop down below.